eng/pipelines/create-doc-index.yml

```diff
@@ -1,12 +1,5 @@
 trigger: none
 
-resources:
```

Comment:
Can we also please do a pass over all the yml files in the branch to remove all the repository references?

Reply:
done